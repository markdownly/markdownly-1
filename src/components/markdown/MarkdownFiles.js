import React from 'react';
import PropTypes from 'prop-types';

export default function MarkdownFiles({ titles }) {
  const titleList = titles.map((title) => {
    return <li key={title}>{title}</li>;
  });
  return (
    <ul>{titleList}</ul>
  );
}

MarkdownFiles.propTypes = {
  titles: PropTypes.array.isRequired
};
