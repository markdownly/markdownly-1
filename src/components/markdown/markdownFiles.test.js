import React from 'react';
import renderer from 'react-test-renderer';
import MarkdownFiles from './MarkdownFiles';

describe('MarkdownFiles', () => {
  it('matches a snapshot', () => {
    const titles = ['file1', 'file2'];

    const tree = renderer.create(
      <MarkdownFiles titles={titles} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
