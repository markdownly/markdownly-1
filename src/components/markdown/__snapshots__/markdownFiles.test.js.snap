// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MarkdownFiles matches a snapshot 1`] = `
<ul>
  <li>
    file1
  </li>
  <li>
    file2
  </li>
</ul>
`;
